src/glib/base/blobbs.h

```diff
@@ -197,12 +197,16 @@ ClassTPV(TBlobBs, PBlobBs, TBlobBsV)//{
 
   void AssertBfCsEqFlCs(const TCs& BfCs, const TCs& FCs);
 
+  // save data stored in SIn for the first time
```

Comment:
Just for the future -- it is good practice to use triple slash for function comments (`///`) since then they are picked up by Doxygen as function description.